easyrsa3/easyrsa

```diff
@@ -88,10 +89,11 @@ cmd_help() {
 
       This request file must exist in the reqs/ dir and have a .req file
       extension. See import-req below for importing reqs from other sources." ;;
-		build|build-client-full|build-server-full) text="
+		build|build-client-full|build-server-full|build-server-client-full) text="
   build-client-full <filename_base> [ cmd-opts ]
   build-server-full <filename_base> [ cmd-opts ]
-      Generate a keypair and sign locally for a client or server
+  build-server-client-full <filename_base> [ cmd-opts ]
+      Generate a keypair and sign locally for a client or server or both
```

Comment:
The help text here could be a bit more verbose, explaining that this option provides both server and client certificate extensions.  This makes it sound like it's still not decided.